docs/src/pages/components/tree-view/ControlledTreeView.js

```diff
@@ -28,7 +28,7 @@ export default function ControlledTreeView() {
 
   return (
     <TreeView
-      aria-label="Controlled TreeView"
+      aria-label="controlled"
```

Comment:
Super important change to deduplicate the "tree"-label here :+1: Otherwise you get announcements like "controlled treeview tree".

Reply:
Screenreader testing ftw :)

Reply:
I need to fix my windows OS. I have a German OS so keywords such as role names are translated which makes redundant names less obvious: "controlled treeview Baumübersicht".